component-samples/manufacturer/src/main/java/org/fidoalliance/fdo/sample/ManufacturerContextListener.java

```diff
@@ -227,9 +193,21 @@ protected void failed(Exception e) {
     DiDbStorage db = new DiDbStorage(cs, ds, keyResolver, ods);
     DiDbManager manager = new DiDbManager();
     manager.createTables(ds);
-    manager.addCustomer(ds, 1, "owner", ownerKeysPem);
-    manager.addCustomer(ds, 2, "reseller", resellerKeysPem);
-    manager.setAutoEnroll(ds, 1);
+    try {
```

Comment:
You can combine both the key additions in the same try block.

Also, the log message for successful/failed addition should be as below.

"Registered public keys for customer 'owner'"
"Registered public keys for customer 'reseller'"

"No default public keys found for customer 'owner'"
"No default public keys found for customer 'reseller'"

Reply:
Done

Reply:
Two separate try blocks are used here as Customer `owner` and 'reseller` are independent cases. There can be cases where the owner key is not present and the reseller keys are present, So we if combine them in the same try block, we might miss updating reseller keys due to the exception raised while loading the owner keys.